crates/matrix-sdk-indexeddb/src/state_store/mod.rs

```diff
@@ -101,6 +102,7 @@ mod keys {
 
     pub const ROOM_STATE: &str = "room_state";
     pub const ROOM_INFOS: &str = "room_infos";
+    pub const RAW_ROOM_INFOS: &str = "raw_room_infos";
```

Comment:
I don't think you'd need a new table, in both cases; the justification being that serializing `Raw<T>` with serde_json is the same as serializing `T` with serde_json. So you should be able to reuse the same table.